### PR TITLE
Examples update for gas-solid fluidized and spouted-bed

### PR DESCRIPTION
**Pull request by OresteMarquis**
### Description

Update of the .prm and .rst files of the gas-solid fluidized and spouted-bed examples for the Lethe 1.0 release.

### Checklist (will be removed when merged)
See [this page](https://chaos-polymtl.github.io/lethe/documentation/contributing.html#pull-requests) for more information about the pull request process.

Code related list:
- [x] Lethe documentation is up to date
- [x] Copyright headers are present and up to date
- [x] The branch is rebased onto master
- [x] Code is indented with indent-all and .prm files (examples and tests) with prm-indent
- [x] Links are added to parent .rst files
- [x] The example is following the [standard format](https://chaos-polymtl.github.io/lethe/documentation/contributing.html#general-rules-and-format)

Pull request related list:
- [x] Labels are applied
- [x] There are at least 2 reviewers (or 1 if small feature) excluding the responsible for the merge
- [x] If this PR closes an issue or is related to a project, it is linked in the "Projects" or "Development" section
- [x] If any future works is planned, an issue is opened
- [x] The PR description is cleaned and ready for merge

### CODE REVIEW

**Review by blaisb — CHANGES_REQUESTED** [review]
Some comments there and there. A generic piece of advice @OresteMarquis is to put as little parameters as possible in the prm file. If the parameter is not modified, then don't put it in the paramter file, the default value will be fine :)

File: examples/unresolved-cfd-dem/gas-solid-fluidized-bed/gas-solid-fluidized-bed.prm [hunk 1]
```diff
@@ -62,17 +63,23 @@ subsection void fraction
   set l2 smoothing factor = 0.000005
   set l2 lower bound      = 0
   set l2 upper bound      = 1
-  set bound void fraction = false
+  set bound void fraction = true
```
[the remark this answers is not in the document]
Comment (blaisb) [reply examples/unresolved-cfd-dem/gas-solid-fluidized-bed/gas-solid-fluidized-bed.prm]: I would change both to false.
It's better to trust the prm file and not the rst file for the parameters.

**Comment by blaisb** [comment]
> How can I know what is the default, so I know what can be removed? I haven't used Lethe enough to know them.

The default values are in the prm template that can be generated with the template parameter executable. I'll show you later if you want.

File: examples/unresolved-cfd-dem/gas-solid-fluidized-bed/gas-solid-fluidized-bed.prm (hunk 1 above)
[the remark this answers is not in the document]
Comment (OresteMarquis) [reply examples/unresolved-cfd-dem/gas-solid-fluidized-bed/gas-solid-fluidized-bed.prm]: Good to know, I changed it to false everywhere!

**Review by blaisb — APPROVED** [review]

**Pull request merged**